TESTS/mbedmicro-rtos-mbed/heap_and_stack/main.cpp

```diff
@@ -204,8 +204,9 @@ void test_isr_stack_in_range(void)
 {
     // MSP stack should be very near end (test using within 128 bytes)
     uint32_t msp = __get_MSP();
-    bool result = inrange(msp, mbed_stack_isr_start + mbed_stack_isr_size - 128, 128);
-
+    printf ("mbed_stack_isr_start = 0x%x mbed_stack_isr_size = 0x%x\n", mbed_stack_isr_start, mbed_stack_isr_size);
+    bool result = inrange(msp, mbed_stack_isr_start, mbed_stack_isr_size);
```

Comment:
I don't understand why a PR tagged as STM needs to update tests?

Reply:
Ohh this change in test should already be on master now.